src/abstractdataframe/join.jl

```diff
@@ -76,21 +89,127 @@ function join_idx(left, right, max_groups)
      right_sorter[right_indexer], right_sorter[rightonly_indexer])
 end
 
-function DataArrays.PooledDataVecs(df1::AbstractDataFrame,
-                                   df2::AbstractDataFrame)
+# FIXME: rename this, and possibly move to CategoricalArrays.jl
+function PooledDataVecs{S,N}(v1::NullableCategoricalArray{S,N},
```

Comment:
Yeah, I think it would be good to rename this as it's clearly a reference to PooledDataArrays. The operation here is a join!/union!/merge!, right?